patitasuy/src/app/pages/chat/chat.page.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { FormsModule } from '@angular/forms';
+import { IonicModule } from '@ionic/angular';
+
+@Component({
+  selector: 'app-chat',
+  templateUrl: './chat.page.html',
+  styleUrls: ['./chat.page.scss'],
+  standalone: true,
```

Comment:
Could you do the move from standalone to ng modules? I've been trying to do so but haven't had any luck. The steps I took to do so, that  i think were succesful were:
- Remove `standalone: true` from every standalone component/page
- Set "type" to "angular" instead of "angular-standalone" in `ionic.config.json` file